editor/src/clj/editor/ios_deploy.clj

```diff
@@ -40,7 +41,7 @@
 (defn- json-stream [input & {:as json-opts}]
   (reify IReduceInit
     (reduce [_ f init]
-      (with-open [reader (io/reader input)]
+      (with-open [reader (PushbackReader. (io/reader input) 64)]
```

Comment:
The pushback buffer size has to be at least 64 chars. The new `data.json` lib version has some performance optimizations for faster string reading. This is only needed for repeated JSON reads from a stream, which seems to be the only place where we do it in the editor.